### ISSUE TITLE
Split out providers into sub packages

### ISSUE THREAD

**Issue by zephraph** [prisma-labs/kont#21]
#### Perceived Problem

Shipping providers inside the main package means consumers pay the cost of the dependencies of those providers even if they're not using said provider. For some things it's may not be a big deal, but for dependencies like playwright it absolutely makes a difference. 

#### Ideas / Proposed Solution(s)

Break the library down into a monorepo. Ship providers as stand along packages in a shared scoped namespace on npm. This will allow supporting things like playwright (and beyond) while not incurring the dependency cost for consumers that don't need it. Having it in one repo but published as different packages means the community can also contribute back to the main repo when adding other useful providers without consumers paying the cost of a new provider. 

---

The biggest challenge with this specific approach is `dripip` doesn't seem to support monorepos. Setting up automatic deployments of monorepos is non-trivial. Historically I've leaned heavily into using [auto](https://github.com/intuit/auto). There is a [conventional commits](https://intuit.github.io/auto/docs/generated/conventional-commits) plugin to map to the current usage. 

Obviously that's one potentially solution among many.

**Comment by jasonkuhrt** [comment]
Agreed with this in principal. Not sure when we'll do it. I might also go a slightly different way. A mega package that bundles all deps. For users that care, letting them install `@kont/core @kont/provider-foo @kont/provider-bar ...`.

![CleanShot 2021-10-04 at 16 30 52@2x](https://user-images.githubusercontent.com/284476/135920661-c26ba2b8-e324-4937-ae6e-d2d7a38d6170.png)

Not that bad. TypeScript is twice that for example.

**Comment by jasonkuhrt** [comment]
But it does seem to drag down CI install times 

<img width="1015" alt="CleanShot 2022-05-03 at 13 40 43@2x" src="https://user-images.githubusercontent.com/284476/166508834-30fc4adf-711e-4372-a3bb-e08c93e33dbb.png">
